### PR TITLE
Cleanup config and get rid of cruft

### PR DESCRIPTION
**Pull request by rall**
This updates the dynamodb config to make it easier to read and keep everything in the same place. We might eventually want a separate production config but this will do for now.

Also removes some cruft cleverly spotted by @waysidekoi

### CODE REVIEW

**Comment by rall** [comment]
forgot, this also removes an extra weird submodule that we don't want

File: config/dynamoid.yml.copy [hunk 1]
```diff
@@ -0,0 +1,19 @@
+shared: &shared
```
Comment (waysidekoi) [inline config/dynamoid.yml.copy]: do we need this file?

Comment (rall) [inline config/dynamoid.yml.copy]: I meants to remove config/dynamoid.yml. We definitely don't want both

Comment (rall) [inline config/dynamoid.yml.copy]: which one gets your vote for the hook?

Comment (waysidekoi) [inline config/dynamoid.yml.copy]: This looks identical to dynamoid.yml -- lets use dynamoid.yml and drop this?

**Pull request merged**